requirements.txt

```diff
@@ -1,3 +1,3 @@
-importlib-resources==1.5.0
+importlib-resources==5.4.0
```

Comment:
this requirement works (as in `make test` passes) now on both internal and public PyPI.